resources/profiles/Geeetech/filament/Generic ASA @Geeetech.json

```diff
@@ -0,0 +1,17 @@
+{
+    "type": "filament",
+    "filament_id": "GFB98",
+    "setting_id": "GFSA04",
+    "name": "Generic ASA @Geeetech",
+    "from": "system",
+    "instantiation": "true",
+    "inherits": "fdm_filament_asa",
+    "filament_flow_ratio": [
+        "0.926"
+    ],
+    "filament_max_volumetric_speed": [
+        "12"
+    ],
+    "compatible_printers": [
+    ]
```

Comment:
the compatible printers need be written, or you could not used this filament in Studio

Reply:
Compatible printers added